db/migrate/20150513034751_add_repository_branches.rb

```diff
@@ -0,0 +1,26 @@
+class AddRepositoryBranches < ActiveRecord::Migration
+  def up
+    create_table :katello_repository_branches do |t|
+      t.string :branch, :null => false
```

Comment:
Ambivalent about this ..  So that column will have data that looks like "redhat-atomic-host/el7.0/x86_64/base"  or "redhat-atomic-host/el7.0/x86_64/medium" 

Ok with name/path/branch -> you get to choose